app/helpers/playbook/pb_doc_helper.rb

```diff
@@ -118,14 +118,18 @@ def get_class_name(kit)
       "Playbook::Pb#{folder.camelize}::#{item.camelize}".safe_constantize
     end
 
-    def render_clickable_title(kit)
+    def render_clickable_title(kit, type)
       url = "#"
       begin
-        url = kit_show_path(kit)
+        url = if type == "react"
```

Comment:
I don't really like this type of assignment here. It is a bit illegible. Could we instead do the assignment within the condition block?

Reply:
rubocop auto fixed it. Let me try it again